core/src/main/java/org/apache/iceberg/util/SnapshotUtil.java

```diff
@@ -149,8 +149,7 @@ public static Iterable<Snapshot> ancestorsOf(long snapshotId, Function<Long, Sna
   }
 
   /**
-   * Traverses the history of the table's current snapshot and finds the first snapshot committed
-   * after the given time.
+   * Finds the oldest snapshot that was committed either at or after a given time.
```

Comment:
Nit: is it necessary to change whole sentence?  How about just replacing 'after' to 'at or after'?

Reply:
The first part of the comment is confusing, it should be at least something like this. 
```
Traverses the table's snapshot history
```
Moreover, I think it is not necessary to mention that. WDYT?

Reply:
Rephrased a bit. Please take a look. Any suggestion is welcome.